### ISSUE TITLE
Error at install skia-safe

### ISSUE THREAD

**Issue by jorgeajimenezl** [rust-skia/rust-skia#728]
Hi
When I try to add skia-safe and make `cargo build`, I got this:
```
Compiling skia-bindings v0.56.0
error: failed to run custom build command for `skia-bindings v0.56.0`

Caused by:
  process didn't exit successfully: `/home/jorgeajimenezl/Documents/repos/test-skia-rs/target/debug/build/skia-bindings-aed19febdaf168e4/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=DOCS_RS
  cargo:rerun-if-env-changed=SKIA_DEBUG
  HOST: x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=SKIA_SOURCE_DIR
  cargo:rerun-if-env-changed=FORCE_SKIA_BUILD
  cargo:rerun-if-env-changed=FORCE_SKIA_BINARIES_DOWNLOAD
  STARTING A FULL BUILD
  cargo:rerun-if-env-changed=CLANGCC
  cargo:rerun-if-env-changed=CC
  cargo:rerun-if-env-changed=CLANGCXX
  cargo:rerun-if-env-changed=CXX
  HOST: x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=OPT_LEVEL
  cargo:rerun-if-env-changed=SKIA_USE_SYSTEM_LIBRARIES
  cargo:rerun-if-env-changed=SDKTARGETSYSROOT
  cargo:rerun-if-env-changed=SDKROOT
  HOST: x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=SKIA_NINJA_COMMAND
  cargo:rerun-if-env-changed=SKIA_GN_COMMAND
  Probing 'python'
  Python 3 found: "python"
  Synchronizing Skia dependencies

  --- stderr
  fatal: not a git repository (or any of the parent directories): .git
  thread 'main' panicked at '`git submodule update` failed', /home/jorgeajimenezl/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.56.0/build_support/binary_cache/download.rs:23:5
```

**Comment by LaoLittle** [comment]
same with me

--- stderr
  fatal: not a git repository (or any of the parent directories): .git
  thread 'main' panicked at '`git submodule update` failed', /Users/laolittle/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.56.0/build_support/binary_cache/download.rs:23:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by tronical** [comment]
I have the same issue when trying to upgrade from 0.55 to 0.56, with a feature set that previously would download the binaries.

**Comment by pragmatrix** [comment]
Caused in #724, which uses the flag `--allow-dirty` for packaging skia-bindings causing the packager to not include `.cargo_vcs_info.json` that is needed in the skia-bindings build script for recognizing that it runs inside a downloaded crate.

Preparing a new release, 0.56.1.

**Comment by pragmatrix** [comment]
Just released version 0.56.1, which - in my tests - fixes the crate detection: https://github.com/rust-skia/rust-skia/releases/tag/0.56.1

Will also add a regression test for that later, so that this particular error can not happen before a release is published.

I am sorry for the inconvenience, closing.

**Comment by tronical** [comment]
Thank you for the quick fix! I can confirm that it works.
